exporters/stats/stackdriver/README.md

```diff
@@ -67,6 +67,26 @@ public class MyMainClass {
 }
 ```
 
+#### Set Monitored Resource for exporter
+
+By default, the Stackdriver Stats Exporter uses [a global Stackdriver monitored resource with no 
+labels](https://cloud.google.com/monitoring/api/resources#tag_global). If you want to use a custom Monitored Resource, do:
+
+```java
+public class MyMainClass {
+  public static void main(String[] args) {
+    // Set a custom MonitoredResource. Please make sure each Stackdriver Stats Exporter has a 
+    // unique MonitoredResource.
+    StackdriverStatsExporter.setMonitoredResource(myResource);
```

Comment:
setMonitoredResource is package protected so you cannot do this.

Reply:
Updated since I've removed this API.